pkg/reconciler/common/transformers.go

```diff
@@ -309,73 +309,99 @@ func SplitsByEqual(arg string) ([]string, bool) {
 }
 
 // TaskImages replaces step and params images.
-func TaskImages(images map[string]string) mf.Transformer {
+func TaskImages(ctx context.Context, images map[string]string) mf.Transformer {
```

Comment:
why we want to make it complex, better to image a new func for stepActions images